CHANGELOG.md

```diff
@@ -39,6 +39,9 @@ Attention: The newest changes should be on top -->
 
 ### Changed
 
+- DOC: Update examples to use `FlightDataExporter` and add deprecation notes for legacy `Flight.export_*` methods. (#XXXX)
+- DEP: `Flight.export_data`, `Flight.export_pressures`, `Flight.export_sensor_data`, and `Flight.export_kml` are deprecated. Use `rocketpy.simulation.flight_data_exporter.FlightDataExporter.export_*`. Planned removal in v1.12.0. (#XXXX)
+- MNT: Extract `Flight.export_data`, `export_pressures`, `export_sensor_data`, and `export_kml` into `rocketpy.simulation.flight_data_exporter.FlightDataExporter` (no behavior change). (#XXXX)
```

Comment:
```suggestion
- MNT: extract flight data exporters [#845](https://github.com/RocketPy-Team/RocketPy/pull/845)
```

This follows the pattern in this repo.

Reply:
[d410bdb](https://github.com/RocketPy-Team/RocketPy/pull/845/commits/d410bdb935f56ea435f412aff6c70ec15aefb50f)